src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/parameters/Base64EncodedStringParameterValue/Base64EncodedStringParameterValue.jelly

```diff
@@ -27,6 +27,9 @@ THE SOFTWARE.
         xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form"
         xmlns:i="jelly:fmt" xmlns:p="/lib/hudson/project">
         <f:entry title="${it.name}" description="${it.description}">
+            <div name="parameter" description="${it.description}">
```

Comment:
I found the issue.

I created `value.jelly` with the below contents on this patch.
```
<?jelly escape-by-default='true'?>
<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define"
        xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form"
        xmlns:i="jelly:fmt" xmlns:p="/lib/hudson/project">
        <f:entry title="Test ${it.name}" description="${it.description}">
            <input type="hidden" name="name" value="${it.name}" />
            <f:textbox name="value" value="${it.value}" readonly="true" />
        </f:entry>
</j:jelly>
```
You can see the string `Test` in the value of `title` attribute for `f:entry` element.

On Jenkins with built plugin, you could see `GERRIT_CHANGE_COMMIT_MESSAGE` like below at parameters page in triggered build.
![value-jelly-cut2](https://cloud.githubusercontent.com/assets/44791/6208343/8ff09ecc-b5fa-11e4-932a-b02254777a10.png)
It means that renamed jelly file has not been used for original purpose since this patch. Perhaps core can display it as of now because core has fallback solution that displays it as `StringParameterValue`.